kin-sdk-core/src/main/java/kin/sdk/core/AccountActivator.java

```diff
@@ -0,0 +1,62 @@
+package kin.sdk.core;
+
+import java.io.IOException;
+import kin.sdk.core.ServiceProvider.KinAsset;
+import kin.sdk.core.exception.AccountNotFoundException;
+import kin.sdk.core.exception.OperationFailedException;
+import org.stellar.sdk.ChangeTrustOperation;
+import org.stellar.sdk.KeyPair;
+import org.stellar.sdk.Server;
+import org.stellar.sdk.Transaction;
+import org.stellar.sdk.responses.AccountResponse;
+import org.stellar.sdk.responses.HttpResponseException;
+import org.stellar.sdk.responses.SubmitTransactionResponse;
+
+class AccountActivator {
+
+    //unlimited trust, The largest amount unit possible in Stellar
+    //see https://www.stellar.org/developers/guides/concepts/assets.html
+    private static final String TRUST_NO_LIMIT_VALUE = "922337203685.4775807";
```

Comment:
do we need this value? I think that if no value provided it use stellar default anyway

Reply:
it doesn't, it's just how the Stellar laboratory expose this, if you look at the generated XDR this value will be generated.
the same goes with the Java SDK, I cannot send empty string, I must provide value myself.

Reply:
cool